extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.items.js

```diff
@@ -107,15 +107,15 @@ angular.module('PaperUI.controllers.configuration').controller('ItemSetupControl
     }
 
     function setItemToFunction() {
-        if ($scope.item['function'].name.indexOf("None") == -1) {
+        if ($scope.item.groupType.indexOf("None") == -1) {
```

Comment:
Do you check that the groupType is not "None"?
What is the intention of this check?